code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -21,10 +21,10 @@
 	var/display_contents_with_number	//Set this to make the storage item group contents of the same type and display them as a number.
 	var/allow_quick_empty	//Set this variable to allow the object to have the 'empty' verb, which dumps all the contents on the floor.
 	var/allow_quick_gather	//Set this variable to allow the object to have the 'toggle mode' verb, which quickly collects all items from a tile.
-	var/collection_mode = 1;  //0 = pick one at a time, 1 = pick all on tile
+	var/collection_mode = TRUE  //FALSE = pick one at a time, TRUE = pick all on tile
```

Comment:
If youre going to make this `TRUE/FALSE` can you rename the var to something else?
`collection_mode = TRUE` does not make sense here